tests/e2e-instrumentation/instrumentation-apache-httpd/chainsaw-test.yaml

```diff
@@ -17,6 +18,7 @@ spec:
         - openshift.io/sa.scc.uid-range=1000/1000
         - --overwrite
     - command:
+        timeout: 10s
```

Comment:
I believe that instead of setting this individually, you can instead set `timeouts.exec` in `.chainsaw.yml`.